circle.yml

```diff
@@ -9,7 +9,7 @@ version: 2
 jobs:
   test:
     machine:
-      docker_layer_caching: true
+      image: ubuntu-1604:201903-01
```

Comment:
May be worthwhile to try and get this back via...

```    machine:
      # We need to specify an image here so we can use docker-compose version 3.4...!
      image: circleci/classic:201808-01
      docker_layer_caching: true
    resource_class: large```

Reply:
:( couldn't figure out how to get docker_layer_caching back...